src/views/Borrow/BorrowTable.tsx

```diff
@@ -68,15 +68,22 @@ export const BorrowTable: React.FC<{ activeType: string }> = () => {
 
   const reserves = useAllReserveTokensWithData();
   const assetRecords = React.useMemo(() => {
-    return (
+    const assets =
       reserves.data?.map(
         ({ symbol, tokenAddress, aTokenAddress }): AssetRecord => ({
           symbol,
           tokenAddress,
           aTokenAddress,
         })
-      ) ?? []
-    );
+      ) ?? [];
+    return assets.map(asset => {
```

Comment:
Use expression lambdas instead of block lambdas in object reconstruction for legibility in the future.